packages/inter-protocol/src/auction/params.js

```diff
@@ -0,0 +1,220 @@
+export const AUCTION_START_DELAY = 'AuctionStartDelay';
+// Basis Points to charge in penalty against vaults that are liquidated.  Notice
+// that if the penalty is less than the LOWEST_RATE discount, vault holders
+// could buy their assets back at an advantageous price.
+export const LIQUIDATION_PENALTY = 'LiquidationPenalty';
+
+// /////// used by VaultDirector /////////////////////
+// time before each auction that the prices are locked.
+export const PRICE_LOCK_PERIOD = 'PriceLockPeriod';
+
+const RelativeTimePattern = { relValue: M.nat(), timerBrand: TimerBrandShape };
+
+export const auctioneerParamPattern = M.splitRecord({
+  [CONTRACT_ELECTORATE]: InvitationShape,
+  [START_FREQUENCY]: RelativeTimePattern,
+  [CLOCK_STEP]: RelativeTimePattern,
+  [STARTING_RATE]: M.nat(),
+  [LOWEST_RATE]: M.nat(),
+  [DISCOUNT_STEP]: M.nat(),
+  [AUCTION_START_DELAY]: RelativeTimePattern,
+  [PRICE_LOCK_PERIOD]: RelativeTimePattern,
+});
+
+export const auctioneerParamTypes = harden({
+  [CONTRACT_ELECTORATE]: ParamTypes.INVITATION,
+  [START_FREQUENCY]: ParamTypes.RELATIVE_TIME,
+  [CLOCK_STEP]: ParamTypes.RELATIVE_TIME,
+  [STARTING_RATE]: ParamTypes.NAT,
+  [LOWEST_RATE]: ParamTypes.NAT,
+  [DISCOUNT_STEP]: ParamTypes.NAT,
+  [AUCTION_START_DELAY]: ParamTypes.RELATIVE_TIME,
+  [PRICE_LOCK_PERIOD]: ParamTypes.RELATIVE_TIME,
+});
+
+/**
+ * @param {object} initial
+ * @param {Amount} initial.electorateInvitationAmount
+ * @param {RelativeTime} initial.startFreq
+ * @param {RelativeTime} initial.clockStep
+ * @param {bigint} initial.startingRate
+ * @param {bigint} initial.lowestRate
+ * @param {bigint} initial.discountStep
+ * @param {RelativeTime} initial.auctionStartDelay
+ * @param {RelativeTime} initial.priceLockPeriod
+ * @param {import('@agoric/time/src/types').TimerBrand} initial.timerBrand
+ */
+export const makeAuctioneerParams = ({
+  electorateInvitationAmount,
+  startFreq,
+  clockStep,
+  lowestRate,
+  startingRate,
+  discountStep,
+  auctionStartDelay,
+  priceLockPeriod,
+  timerBrand,
+}) => {
+  return harden({
+    [CONTRACT_ELECTORATE]: {
+      type: ParamTypes.INVITATION,
+      value: electorateInvitationAmount,
+    },
+    [START_FREQUENCY]: {
+      type: ParamTypes.RELATIVE_TIME,
+      value: TimeMath.toRel(startFreq, timerBrand),
+    },
+    [CLOCK_STEP]: {
+      type: ParamTypes.RELATIVE_TIME,
+      value: TimeMath.toRel(clockStep, timerBrand),
+    },
+    [AUCTION_START_DELAY]: {
+      type: ParamTypes.RELATIVE_TIME,
+      value: TimeMath.toRel(auctionStartDelay, timerBrand),
+    },
+    [PRICE_LOCK_PERIOD]: {
+      type: ParamTypes.RELATIVE_TIME,
+      value: TimeMath.toRel(priceLockPeriod, timerBrand),
+    },
+    [STARTING_RATE]: { type: ParamTypes.NAT, value: startingRate },
+    [LOWEST_RATE]: { type: ParamTypes.NAT, value: lowestRate },
+    [DISCOUNT_STEP]: { type: ParamTypes.NAT, value: discountStep },
+  });
+};
+harden(makeAuctioneerParams);
+
+export const toParamValueMap = typedDescriptions => {
```

Comment:
this looks like it was meant to be `td => td.value`.

That said, I doubt the value of this helper as `objectMap(typedDescriptions, td => td.value)` is quite concise